pkg/kubelet/cm/util/constants.go

```diff
@@ -0,0 +1,5 @@
+package util
+
+import "time"
+
+const PluginTimeout = 10 * time.Second
```

Comment:
Creating a new file just for one constant doesn't look optimal to me. Let's do it the same way as it's done [in the device manager](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/devicemanager/plugin/v1beta1/client.go#L124).

I was hoping to find a place where to put it and share between DRA and Device plugins, but it looks like a common practice is to use local constants or hardcoded timeouts.

Reply:
I think it's still worth having a constant -- just put the constant at the top of `pkg/kubelet/cm/dra/plugin/client.go` instead of `pkg/kubelet/cm/util/constants.go`.

Reply:
@Saza-ku can you address @klueska's comment, please?